### ISSUE TITLE
Documentation - How should the changes logs be read in the official IFC4X3_ADD2 documentation?

### ISSUE THREAD

**Issue by gmn3DS** [buildingSMART/IFC4.x-IF#142]
Hello everybody,

The title is quite self-explanatory but I am having trouble understanding the [change logs](https://standards.buildingsmart.org/IFC/RELEASE/IFC4_3/HTML/annex-f.html#IFC4.2).
Against which prior version should they be regarded?

( I think my confusion might stem from the presence of IFC 4.1 and IFC 4.2 logs since both schema have been withdrawn. )

Best regards,
Guillaume

**Comment by aothms** [comment]
Within the sequence of schema IFC2X3_TC1 IFC4_ADD2_TC1 IFC4X1 IFC4X2 IFC4X3_ADD2 the change logs are given relative to the previous in this list.

You can use https://github.com/buildingSMART/IFC4.3.x-development/blob/master/code/change_log.py to generate a changelog yourself for a schema pair of interest.

**Comment by gmn3DS** [comment]
Hello @aothms,

So I spent some time trying to use change_log.py, without success.
For the record (Python 3.11.1 win10-64), I realized the following steps:

- Cloned the repository **IFC4.3.x-development** and replaced content from (same) zipped repository. (It solved one of my issues)
- Install the *requirements* (in a python virtual environment)
- Install *ifcopenshell* pre-built package according to my configuration
- Install *lark* library because it seemed to be missing from the requirements

I am currently struggling with the following error:
![image](https://github.com/buildingSMART/IFC4.x-IF/assets/124370855/81254d20-0adb-4744-b016-709d16faf572)

More than me overcoming the environment setup, I guess it would be natural to have change logs between official ISO schema. 
My suggestion is to add change logs between **IFC4_ADD2_TC1** and **IFC4X3_ADD2**  to the unofficial documentation, would that be possible? 

Thanks,
Guillaume
